modules/api/php/projectinstrumentsrow.class.inc

```diff
@@ -0,0 +1,115 @@
+<?php declare(strict_types=1);
+/**
+ * This class implements a data Instance for project's instruments.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api;
+
+/**
+ * A ProjectInstrumentsRow represents an instrument summary for a given project.
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class ProjectInstrumentsRow implements \LORIS\Data\DataInstance
+{
+    private $_shortname;
+    private $_fullname;
+    private $_subgroupname;
+    private $_isDDE;
+
+    /**
+     * Create a new ProjectImagesRow.
+     *
+     * @param array $row An array of image properties
+     */
+    public function __construct(array $row)
+    {
+        $shortname      = $row['shortname'] ?? null;
+        $ddeinstruments = array_keys(\Utility::getAllDDEInstruments());
+
+        $this->_shortname    = $shortname;
+        $this->_fullname     = $row['fullname'] ?? null;
+        $this->_subgroupname = $row['subgroupname'] ?? null;
+        $this->_isDDE        = in_array(
+            $shortname,
+            $ddeinstruments
```

Comment:
```suggestion
            $ddeinstruments,
            true
```

In general it's better for `in_array` to have the true parameter or else weird things can happen.